.github/workflows/dispatch-build-bottle.yml

```
@@ -145,19 +145,19 @@ jobs:
         env:
           HOMEBREW_BINTRAY_USER: brewtestbot
           HOMEBREW_BINTRAY_KEY: ${{secrets.HOMEBREW_BINTRAY_KEY}}
-          HOMEBREW_GPG_PASSPHRASE: ${{ secrets.BREWTESTBOT_GPG_SIGNING_SUBKEY_PASSPHRASE }}
-          BREWTESTBOT_NAME_EMAIL: "BrewTestBot <1589480+BrewTestBot@users.noreply.github.com>"
         run: |
           cd ~/bottles
-          brew pr-upload --verbose --keep-old --committer="$BREWTESTBOT_NAME_EMAIL"
+          brew pr-upload --verbose --keep-old --no-commit --root-url="https://ghcr.io/v2/homebrew/core"
 
       - name: Upload bottles to GitHub Packages
         env:
           HOMEBREW_GITHUB_PACKAGES_USER: brewtestbot
           HOMEBREW_GITHUB_PACKAGES_TOKEN: ${{secrets.HOMEBREW_CORE_GITHUB_PACKAGES_TOKEN}}
+          HOMEBREW_GPG_PASSPHRASE: ${{ secrets.BREWTESTBOT_GPG_SIGNING_SUBKEY_PASSPHRASE }}
+          BREWTESTBOT_NAME_EMAIL: "BrewTestBot <1589480+BrewTestBot@users.noreply.github.com>"
         run: |
           cd ~/bottles
-          brew pr-upload --verbose --keep-old --no-commit --root-url="https://ghcr.io/v2/homebrew/core"
+          brew pr-upload --verbose --keep-old --committer="$BREWTESTBOT_NAME_EMAIL"
```

Need `--root-url` here. That's what the problem was.

```suggestion
          brew pr-upload --verbose --keep-old --no-commit --root-url="https://homebrew.bintray.com/bottles"
```

---

Should we do the same in `publish-commit-bottles.yml`?

---

Yes.